apps/docs/content/guides/platform/read-replicas.mdx

```diff
@@ -102,6 +102,15 @@ To call a read-only Postgres function on Read Replicas through the REST API, use
 
 If you remove all Read Replicas from your project, the load balancer and its endpoint are removed as well. Make sure to redirect requests back to your Primary database before removal.
 
+#### Experimental Routing
+
+The API load balancer offers an experimental routing mode that builds on top of the existing funtionality. It is now possible to use your load balancer endpoint for all Supabase related traffic. The following list summarizes the features that are now accessible.
+
+- Auth requests are sent to the Primary database even when using the load balancer endpoint.
+- Routing of Edge Functions, Storage, and Realtime requests go to the appropriate backend Supabase service.
```

Comment:
While internally this is a routing problem in terms of implementation, from the user's perspective it's really more of a "The LB endpoint now supports usage of additional services (Auth, Storage, Realtime, Edge Functions)."

Can add a note that "Due to the requirements of the Auth service, all Auth requests are handled by the Primary, even when sent over the LB endpoint, similar to how non-Read requests for the Data API (Postgrest) are exclusively handled by the Primary."

Reply:
Updated.